functions/src/callable.ts

```diff
@@ -1,14 +1,39 @@
-import { HttpsError, onCall } from "firebase-functions/v2/https";
-import { getTTS } from "./deepgram";
-
-export const tts = onCall(async (request) => {
-  try {
-    const { text } = request.data;
-    return await getTTS(text);
-  } catch (error) {
-    throw new HttpsError(
-      "unknown",
-      "Something went wrong while trying to fetch tts"
-    );
-  }
-});
+// import { HttpsError, onCall } from "firebase-functions/v2/https";
+// import { getTTS } from "./deepgram";
+// import { chunkTextDynamically } from "./utils";
+
+// export const tts = onCall(async (request, response) => {
+//   try {
+//     let { text } = request.data;
+
+//     text = text
+//     .replaceAll("¡", "")
+//     .replaceAll("https://", "")
+//     .replaceAll("http://", "")
+//     .replaceAll(".com", " dot com")
+//     .replaceAll(".org", " dot org")
+//     .replaceAll(".co.uk", " dot co dot UK")
+//     .replaceAll(/```[\s\S]*?```/g, "\nAs shown on the app.\n")
+//     .replaceAll(
+//       /([a-zA-Z0-9])\/([a-zA-Z0-9])/g,
+//       (_: any, precedingText: string, followingText: string) => {
+//         return precedingText + " forward slash " + followingText;
```

Comment:
Delete all that. If we wanna look at it later we will check git history

Reply:
aight